svg/painting/parsing/stroke-computed.svg

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml"
+     width="800px" height="800px">
+  <title>SVG Geometry Properties: getComputedValue().stroke</title>
+  <metadata>
+    <h:link rel="help" href="https://svgwg.org/svg2-draft/painting.html#StrokeProperty"/>
+    <h:meta name="assert" content="stroke computed value is as specified, with url values absolute."/>
+  </metadata>
+  <g id="target"></g>
+  <h:script src="/resources/testharness.js"/>
+  <h:script src="/resources/testharnessreport.js"/>
+  <h:script src="/css/support/computed-testcommon.js"/>
+  <script><![CDATA[
+
+test_computed_value("stroke", "none");
+test_computed_value("stroke", "rgb(12, 34, 56)");
+
+test_computed_value("stroke", 'url("https://example.com/")');
+test_computed_value("stroke", 'url("https://example.com/") none');
+test_computed_value("stroke", 'url("https://example.com/") rgb(12, 34, 56)');
```

Comment:
Might want to test (here and for `fill`) that the `color url()` values get swapped into a canonical order.

Reply:
I think values may only occur in one order - there is no `||` in the `<paint>` [grammar](https://svgwg.org/svg2-draft/painting.html#SpecifyingPaint).

The [stroke-invalid.svg](https://wpt.fyi/results/svg/painting/parsing/stroke-invalid.svg?label=stable&aligned) and [fill-invalid.svg](https://wpt.fyi/results/svg/painting/parsing/fill-invalid.svg?label=stable&aligned) tests reject invalid orders.

Reply:
Interesting. I'd never tested it. Another case where SVG rules aren't following modern CSS practices. But I'm not going to argue for changing it at this point.